src/ssl.c

```diff
@@ -1205,6 +1205,34 @@ int wolfSSL_set_ConnectFilter(
 #endif /* WOLFSSL_WOLFSENTRY_HOOKS */
 
 #ifndef WOLFSSL_LEANPSK
+#if defined(WOLFSSL_WRAPPER_ALLOCATORS) && defined(XINET_PTON) && \
```

Comment:
Do we really need another new macro `WOLFSSL_WRAPPER_ALLOCATORS`? I almost prefer this to be enabled in the cases where sockets and DTLS exist. Is this already wrapped in `WOLFSSL_DTLS`?

Reply:
Not wrapped in `WOLFSSL_DTLS` but I see what you're saying. Two functions don't seem worth a new macro. Will switch to 
```
#if defined(WOLFSSL_DTLS) && defined(XINET_PTON) && \                           
  !defined(WOLFSSL_NO_SOCK) && defined(HAVE_SOCKADDR)
```